DB/CreatingBTdb.sql

```diff
@@ -0,0 +1,171 @@
+/* First of all to create database execute:
```

Comment:
Please add this file to the solution, so that it would be visible in VS.

Reply:
It wasn't a file. It was commented SQL instruction that just should be executed separately of the rest code

Reply:
I mean the file `CreatingBTdb.sql`, add it to the solution. 
![image](https://user-images.githubusercontent.com/66346727/108962932-baadba00-7679-11eb-8452-5d43cb30bae3.png)